src/app/profile/overview/activity/activity.component.html

```diff
@@ -1,33 +0,0 @@
-<div *ngIf="activityItems.length === 0; then showBlankState else showActivityItems"></div>
-<ng-template #showBlankState>
-  <div class="col-sm-12">
-    <div class="blank-slate-pf">
-      <h1>{{context?.user?.attributes?.username}} has no activity</h1>
-      <p>Go to the <a href="javascript:void(0)"
-                      [routerLink]="['/', '_home']">home</a> page to get started in OpenShift.io.</p>
-      <div class="blank-slate-pf-main-action">
-        <button class="btn btn-primary btn-lg"
-                (click)="routeToHome()">Get Started</button>
-      </div>
-    </div>
-  </div>
-</ng-template>
-<ng-template #showActivityItems>
-  <h2 class="margin-left-15">{{context?.user?.attributes?.username}}'s recent activity</h2>
-  <div class="col-sm-12">
-    <ul class="list-group list-view-pf list-view-pf-view">
-      <li class="list-group-item" *ngFor="let activity of activityItems | take: 10">
-        <div class="list-view-pf-main-info">
-          <div class="list-view-pf-body">
-            <div class="list-view-pf-description">
-              <div class="list-group-item-text margin-right-0">
-                <span>{{activity.what}}</span>
-                <span class="pull-right">{{activity.when}}</span>
-              </div>
-            </div>
-          </div>
-        </div>
-      </li>
-    </ul>
-  </div>
-</ng-template>
```

Comment:
Should we delete the activity component? I realize this isn't much, but we will show activity at some point?

Reply:
I deleted it as there isn't anything in the pipeline right now for activity. It is just an empty component that wouldn't be rendered anywhere. I can put it back, but don't see the need right now.